packages/dev/core/src/Materials/Textures/dynamicTexture.ts

```diff
@@ -207,6 +210,19 @@ export class DynamicTexture extends Texture {
         }
     }
 
+    /**
+     * Disposes the dynamic texture.
+     */
+    public dispose(): void {
+        super.dispose();
+
+        if (this._ownCanvas) {
+            this._canvas.remove();
+        }
+        (this._canvas as any) = null;

```

Comment:
Should these go under the same check on line 219? Would we only want to null out `_canvas` and `_context` if we had created those?

Reply:
Nulling is not a problem even if we didn't create the canvas/context. It will only indicate the JVM that we don't need to access these objects anymore, it won't modify the objects themselves.